CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportDelimitedPlugin.java

```diff
@@ -112,6 +113,12 @@ public PluginParameters createParameters() {
         parserParameters.setDescription("The PluginParameters used by the parser");
         params.addParameter(parserParameters);
 
+        final PluginParameter<BooleanParameterValue> filesIncludeHeadersParam = BooleanParameterType.build(FILES_INCLUDE_HEADERS_PARAMETER_ID);
+        filesIncludeHeadersParam.setName("files Include Headers");
```

Comment:
I'd prefer we start with a capitalised word (i.e "Files") but its interesting to see the GUI has it capitalised.

Reply:
Oh my bad, but I guess this is not really used in the GUI (that's why it worked before the changes here) and should be used in the API if we ever expose the parameters.